contracts/utils/Create3Address.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title Create3Address contract
+ * @notice This contract can be used to predict the deterministic deployment address of a contract deployed with the `CREATE3` technique.
+ */
+contract Create3Address {
+    // slither-disable-next-line too-many-digits
+    bytes internal constant CREATE_DEPLOY_BYTECODE =
```

Comment:
add a comment about this being the fixed CreateDeploy contract bytecode being used by ITS, and fixing this allows changes to compiler settings to not affect the address derivation